### ISSUE TITLE
Contribution or Collaboration 

### ISSUE THREAD

**Issue by morpheuslord** [HalilDeniz/PassBreaker#2]
Hello HalilDeniz,

I am a cybersecurity enthusiast and have a project related to cybersecurity, the [GPT-Vuln Analyzer](https://github.com/morpheuslord/GPT_Vuln-analyzer) I would be delighted to include your tool in my toolset, with your permission. If feasible, could you create a pull request with the update? This would allow me to add you as a contributor.

Thank you for considering this,
morpheuslord

**Comment by HalilDeniz** [comment]
Hello. First of all, I would like to thank you for this thoughtful and kind thought and I would be pleased. But since I don't know enough about this subject, can you please tell me what I should do step by step?

**Comment by morpheuslord** [comment]
Great, Thanks for reaching back. I tried reaching out to you via LinkedIn but I made a silly mistake so I am using your issues section for this. As far as steps go:

-  Clone my tool [GPT_Vuln-Analyzer](https://github.com/morpheuslord/GPT_Vuln-analyzer)
-  In the cloned directory add your `PassBreaker.py` file in the `components` directory.
-  In the root directory create a sample code for the usage demo of the `PasswordCracker` class.

Once you have done that create a new PR of the update I will make the necessary changes and add this feature to the GUI and CLI  components of my tool. If you want we can connect via [LinkedIn](https://www.linkedin.com/in/chiranjeevi-g-naidu/) for further discussions.
